src/chapters/more-on-functions/recursion-wrap-up.rst

```diff
@@ -0,0 +1,53 @@
+Recursion Wrap-Up
+==================
+
+In order to function (ba-dum chhhh), recursion must fulfill four conditions:
+
+#. A series of small, identical steps combine to solve a larger problem.
+#. A base case must be defined.  When true, this simplest case halts the
+   recursion.
+#. A recursive function repeatedly calls itself.
+#. Each time the recursive function is called, it must alter the
+   data/variables/conditions in order to move closer to the base case.
+
+Benefits of Recursion:
+
+a. Fewer lines of code required to accomplish a task,
+b. Makes code cleaner and more readable.
+
+Drawbacks of Recursion:
+
+a. More abstract than using loops,
+b. Code is "more readable" only if the reader understands recursion.
+
+Why Do I Need To Know Recursion?
```

Comment:
Nice job on this. I think it will motivate students and contextualize recursion appropriately.

Reply:
Thanks.